en/docs/develop/product-apis/publisher-apis/publisher-v1/publisher-v1.yaml

```diff
@@ -926,7 +942,11 @@ paths:
         GET https://localhost:9443/api/am/publisher/v1/apis/890a4f4d-09eb-4877-a323-57f6ce2ed79b/resource-policies/0bcb7f05-599d-4e1a-adce-5cb89bfe58d5
         Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
       x-wso2-response: "HTTP/1.1 200 OK\nContent-Type: application/json\n\n{\r\n            \"id\": \"6c735e1d-8041-4f1e-b246-d28edaa650b0\",\r\n            \"httpVerb\": \"post\",\r\n            \"resourcePath\": \"checkPhoneNumbers\",\r\n            \"content\": \"<header description=\\\"SOAPAction\\\" name=\\\"SOAPAction\\\" scope=\\\"transport\\\" value=\\\"http://ws.cdyne.com/PhoneVerify/query/CheckPhoneNumber\\\"/>\"\r\n}"
```

Comment:
x-wso2-response is not in use anymore. Shall we remove this?

Reply:
fixed with https://github.com/wso2/docs-apim/pull/1812/commits/ff8d80c40e3adcb3ac46d01dcb950f0d4d164502